crossfit/backend/torch/model.py

```diff
@@ -41,3 +56,41 @@ def estimate_memory(self, max_num_tokens: int, batch_size: int) -> int:
 
     def max_seq_length(self) -> int:
         raise NotImplementedError()
+
+    def get_model_output(self, all_outputs_ls, index, loader, pred_output_col) -> cudf.DataFrame:
+        out = cudf.DataFrame(index=index)
+        _index = (
+            loader.sort_column(index.values)
+            if loader.__class__.__name__ == "SortedSeqLoader"
+            else index
```

Comment:
This is not recommended. Please use isinstace.

Reply:
I have a doubt here, this sorting of index required only for SortedSeqLoader and not for InMemoryLoader, right ?
If this is the case, then isinstance will return True even for InMemoryLoader loader object.

Reply:
```python3
class Base:
    pass

class Derived(Base):
    pass

obj1 = Base()
obj2 = Derived()

# Check for exact instance of Base, not matching subclasses
if type(obj1) is Base:
    print("obj1 is exactly of type Base")

if type(obj2) is Base:
    print("obj2 is exactly of type Base")
else:
    print("obj2 is not exactly of type Base")

# Check using isinstance, which includes subclasses
if isinstance(obj1, Base):
    print("obj1 is an instance of Base or its subclass")

if isinstance(obj2, Base):
    print("obj2 is an instance of Base or its subclass")
 ```
 
 ```python3
 obj1 is exactly of type Base
obj2 is not exactly of type Base
obj1 is an instance of Base or its subclass
obj2 is an instance of Base or its subclass
 ```
 
I think if you want to match subclasses , you can use type instead.

Reply:
Done